lib/transaction.js

```diff
@@ -141,7 +140,7 @@ export default class Transaction {
       offset += ((type !== Type.TRANSFER && type !== Type.CONSOLIDATE) && i === 0) ? 33 : 98;
     }
     raw.copy(noSigs, offset, offset, raw.length);
-    return ethUtil.sha3(noSigs);
+    return ethUtil.keccak256(noSigs);
```

Comment:
why not put `hashPersonalMessage()` here instead of `keccak256()`

then, instead of `keccak(prepend + keccak(txBytes))` we would do `keccak(prepend + txBytes)`, which would maybe enable us to use ERC712 later as @troggy suggested.

Reply:
🤔 yeah, probably would be better.

We're signing hash, so we need to apply `hashPersonalMessage` on hash, not raw data